### ISSUE TITLE
Variant and orthography of test sets

### ISSUE THREAD

**Issue by ftyers** [AmericasNLP/americasnlp2021#11]
Hi, would it be possible to find out what variant and what orthography will be used for the test sets? e.g. the Nahuatl data has both *siua*, *sihua* and *cihua*, but no *siwa*.

**Comment by pywirrarika** [comment]
Hi. The variant and orthography of test and the dev sets are going to be the same. Nahuatl has many variations in the training set. So this is something to take in account.

**Comment by ftyers** [comment]
Aah, ok, I got it, so the test set will match the dev set in orthography/variant. Thanks!

**Comment by pywirrarika** [comment]
YW. Yes. sorry for not being clear. Ops
